src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
```

Comment:
You should link to the CSS file, not the SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document.